.github/workflows/labeler.yml

```diff
@@ -9,3 +9,10 @@ jobs:
         with:
           sync-labels: ''
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
+  milestone:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: iyu/actions-milestone@v1
```

Comment:
I forgot to change it in the `users: actions/labeler@main` line too (already [did it for the needs-rebase workflow](https://github.com/ceph/ceph/blob/master/.github/workflows/needs-rebase.yml#L14)), but due to [security concerns](https://julienrenaux.fr/2019/12/20/github-actions-security-risk/) (as the actions have access to our GH Token), it'be safer if we pin the actions to specific commits (and perhaps we should perform some basic checking in the action source-code to ensure it does not log/pushes the token anywhere).

Reply:
do you want me to add a commit to fix this?

Reply:
the linked article is a very good reading, thank you! I also point the actions-milestone to a sha1 in the same commit.